inventory_management_system_api/services/system.py

```diff
@@ -16,8 +15,6 @@
 from inventory_management_system_api.schemas.system import SystemPatchSchema, SystemPostSchema
 from inventory_management_system_api.services import utils
 
-logger = logging.getLogger()
```

Comment:
I can see that other service modules have this that is unused. We should probably open an issue.

Reply:
I have made a note - I can probably do that along with the minor test fixes I mentioned in the error handling pr when I copy that across.